presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableWriterNode.java

```diff
@@ -70,9 +71,10 @@ public TableWriterNode(
             @JsonProperty("notNullColumnVariables") Set<VariableReferenceExpression> notNullColumnVariables,
             @JsonProperty("partitioningScheme") Optional<PartitioningScheme> tablePartitioningScheme,
             @JsonProperty("preferredShufflePartitioningScheme") Optional<PartitioningScheme> preferredShufflePartitioningScheme,
-            @JsonProperty("statisticsAggregation") Optional<StatisticAggregations> statisticsAggregation)
+            @JsonProperty("statisticsAggregation") Optional<StatisticAggregations> statisticsAggregation,
+            @JsonProperty("taskNumberIfScaledWriter") Optional<Integer> taskNumberIfScaledWriter)
```

Comment:
Add a field to specify the number of tasks to begin with if it's a scaled writer